admin_app/tsconfig.json

```diff
@@ -5,13 +5,13 @@
     "skipLibCheck": true,
     "strict": true,
     "noEmit": true,
-    "incremental": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
I don't understand all the implications of this. Do you guys?

Reply:
I don't either